src/main/java/org/spongepowered/api/effect/Viewer.java

```diff
@@ -161,4 +161,60 @@ default void resetBlockChange(final Vector3i position) {
      */
     void resetBlockChange(int x, int y, int z);
 
+    /**
+     * Sends a client-only block breaking progress.
+     *
+     * <p>In vanilla breaking progress will not be rendered if player
+     * is further than 32 blocks from given position.<br>
+     * Sent breaking progress expires on client 400 ticks after receiving.</p>
+     *
+     * @param position The position
+     * @param progress The breaking progress from 0 to 1 (1 excluded)
+     */
+    default void sendBlockProgress(final Vector3i position, final float progress) {
+        Objects.requireNonNull(position, "position");
+        this.sendBlockProgress(position.x(), position.y(), position.z(), progress);
+    }
+
+    /**
+     * Sends a client-only block breaking progress.
+     *
+     * <p>In vanilla breaking progress will not be rendered if player
+     * is further than 32 blocks from given position.<br>
+     * Sent breaking progress expires on client 400 ticks after receiving.</p>
```

Comment:
Sponge doesn't (and can't) do anything with this because it's client stuff. But for plugin devs this info could be useful so they don't have to go to sources and I'm pretty sure api has a bunch of these `by default` additions.
From what I just found:
https://github.com/SpongePowered/SpongeAPI/blob/22c65048cba23a68f12e83a6cf98225be5ab7088/src/main/java/org/spongepowered/api/data/Keys.java#L742
https://github.com/SpongePowered/SpongeAPI/blob/42269e2110de472a40447c27d0f41a542900432f/src/main/java/org/spongepowered/api/event/cause/entity/damage/DamageType.java#L49
https://github.com/SpongePowered/SpongeAPI/blob/b2216ba4ebe77b96e059441fbd2a067746e948bb/src/main/java/org/spongepowered/api/block/entity/StructureBlock.java#L67